beetsplug/keyfinder.py

```diff
@@ -52,15 +53,19 @@ def imported(self, session, task):
 
     def find_key(self, items, write=False):
         overwrite = self.config['overwrite'].get(bool)
-        bin = self.config['bin'].as_str()
+        command = [self.config['bin'].as_str()]
+        # if using KeyFinder GUI program, the path must be prefixed with -f flag
+        # keyfinder-cli instead wants the path bare
```

Comment:
We tend to use full sentences in beets comments. Here's a suggested way to phrase it:

> The KeyFinder GUI program needs the `-f` flag before the path. keyfinder-cli is similar but just wants the path with no flag.